### PR TITLE
Added support for OkHttp

### PR DESCRIPTION
**Pull request by whiskeysierra**

### CODE REVIEW

File: tracer-core/src/main/java/org/zalando/tracer/StackedTracer.java [hunk 1]
```diff
@@ -89,7 +89,7 @@ public void forEach(final BiConsumer<String, String> consumer) {
     public void stop() {
         traces.forEach((name, state) -> {
             final Deque<String> queue = state.get();
-            final String previous = checkValue(name, queue.removeLast());
+            final String previous = checkValue(name, queue.pollLast());
```
Comment (AlexanderYastrebov) [inline tracer-core/src/main/java/org/zalando/tracer/StackedTracer.java]: If would be nullable now

Comment (whiskeysierra) [reply tracer-core/src/main/java/org/zalando/tracer/StackedTracer.java]: Correct, I fixed it.

File: tracer-okhttp/pom.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            <artifactId>okhttp</artifactId>
+            <version>3.9.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-nop</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.github.rest-driver</groupId>
+            <artifactId>rest-client-driver</artifactId>
```
Comment (AlexanderYastrebov) [inline tracer-okhttp/pom.xml]: scope=test

**Comment by AlexanderYastrebov** [comment]
:+1:

**Pull request merged**